.github/workflows/update-keycloak-dependency.yml

```diff
@@ -95,5 +95,6 @@ jobs:
         run: |
           git config --global user.name "GitHub Actions"
           git config --global user.email "actions@github.com"
+          git checkout ${{ env.BRANCH_NAME }}
           git tag -a "v${{ env.PACKAGE_VERSION }}" -m "Release version ${{ env.PACKAGE_VERSION }}" -f
```

Comment:
should we also have a tag "latest" which can be referenced from server's package.json?